specification/purview/data-plane/Azure.Analytics.Purview.DataQuality/preview/2022-09-01-preview/purviewDataQuality.json

```diff
@@ -0,0 +1,3820 @@
+            "schema": {
+              "$ref": "#/definitions/ViewSourceDetail"
+            }
+          },
+          "default": {
+            "description": "An error response received from the DataQuality Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ViewSource_Get": {
+            "$ref": "./examples/ViewSource_Get.json"
+          }
+        }
+      }
+    },
+    "/accounts/{accountId}/viewsources/{viewSource}/views/{view}/validation": {
+      "head": {
+        "tags": [
+          "DataQuality View Sources"
+        ],
+        "description": "Check View Name is Unique or not in a View Source.",
+        "operationId": "ViewSourceViewName_Validate",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "accountId",
+            "description": "Get Account Id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "viewSource",
+            "description": "Get View Source",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "view",
+            "description": "Get View",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "default": {
+            "description": "An error response received from the DataQuality Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ViewSourceViewName_Validate": {
+            "$ref": "./examples/ViewSourceViewName_Validate.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AlertingScope": {
+      "description": "The Alert Scope Model",
+      "type": "object",
+      "properties": {
+        "level": {
+          "enum": [
+            "CATALOG_PATH"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ScopeLevel",
+            "modelAsString": true
+          }
+        },
+        "type": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        }
```

Comment:
Please include descriptions on all properties. I've only commented here but there are many properties without descriptions -- please fix all of these.